prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	arrowParens: 'avoid',
+	bracketSpacing: true,
+	jsxBracketSameLine: false,
+	parenSpacing: true,
+	printWidth: 100,
```

Comment:
[MINOR] What are your thoughts on increasing this? Maybe to `150`?
I tested with the side panel open + 2 split tabs in a 4k resolution. I could write `150` chars on each tab and there is still `~20` chars left on both panels. I use a 32'' monitor though.

Reply:
I think this setting is less about ergonomics when writing code and more about reviewing it. Someone on a laptop and using GitHub side-by-side diff might like a lower value.